components/Map/Map.js

```diff
@@ -16,12 +15,16 @@ export default function Map({ navigation, events }) {
       }}
       >
       {events.map(event => {
+
+        let eventLat = event.lat_long.split(',')[0];
+        let eventLong = event.lat_long.split(',')[1];
```

Comment:
I'm glad we were able to split up the lat and long into usable data via this code. Way to go!